bigframes/dtypes.py

```diff
@@ -280,6 +281,10 @@ def ibis_dtype_to_bigframes_dtype(
     if isinstance(ibis_dtype, ibis_dtypes.Integer):
         return pd.Int64Dtype()
 
+    # Temporary: Will eventually support an explicit json type instead of casting to string.
```

Comment:
We should probably raise a warning (PreviewWarning?) in this case to make sure folks know that depending on any JSON functionality may break in future.

Reply:
added a preview warning